src/config.ts

```diff
@@ -76,45 +77,66 @@ export const MAP_DATA_ACCEPTED_TYPES_NAMES = [
   'ESRI Shapefile',
 ];
 
+const getTypesExtensions = (types: Record<string, string[]>) =>
+  Object.values(types)
+    .flat()
+    .map(ext => ext.substr(1))
```

Comment:
question: is this to remove the leading `.`?

Reply:
@paulschreiber That is what I understand yes, this change was not added as part of this PR, I just removed the duplication